doc/configuration.rst

```diff
@@ -3084,12 +3084,23 @@ Implements:
 
    DockerDriver:
      image_uri: 'rastasheep/ubuntu-sshd:16.04'
+     pull: 'newer'
```

Comment:
issue: not implemented

`newer` doesn't exist (not implemented in this change), but `never` does, did you mean `never` here?

Reply:
Ah yes, well spotted. Actually a bit worse - I meant to write `always` as it will then be the same behavior as the previous implementation.

Reply:
but if it's already the default, why mention it in the example :) ?

Reply:
I think it is good practice to show the options. I would rather have the users using a function they might not need, than having them overlook it's existence, and therefore spend hours looking through documentation and source code (as I did).

Beside from that, I don't have a strong opinion whether it should be `always` or `never` - you get the final say in this :)

Reply:
So, you get to choose the value, `always`, `missing` or `never` :)?

Reply:
`always` is fine so we don't need to have a new push just for that :)